resources/keywords/import_provider_account.resource

```diff
@@ -119,9 +120,23 @@ User Enters Invalid Data To Import CockroachDB Provider Account
     Click Element    ${btnImportEnabled}
 
 Provider Account Import Success
+    ${no_dbinstance}=    Run Keyword And Return Status    Success Screen For A Provider Account With No DB Instances
```

Comment:
This should be failure and the mitigation for application not showing up the DBaaSinventory screen should be avoided. 
As per the expectation, once the provider account is imported, the application should navigate to success screen and when the user clicks on "View Provider Account" button, The Application should navigate to DBaaS Inventory screen. If not it should be considered as Error and mitigation for the user to navigate to DBaaS inventory should be avoided.

Reply:
In that case the current flow is the application is not as per expectation. Let's discuss this, because this a corner case and we are not mitigating the error, we are handling the error to avoid false failure as the error msg is expected and currently implemented.

Reply:
I agree for the validation you are doing now